.projenrc.js

```diff
@@ -67,7 +67,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     "CONTRIBUTING.md",
   ],
   scripts: {
-    "check-formatting": "prettier --check src/**",
+    "check-formatting": "prettier --check src/** integration_tests/**/*.ts examples/**/*.ts",
```

Comment:
I tried to find a more succinct way to get all `.ts` files in the repo, but couldn't find one. For example, I tried `**/*.ts` and `*/**/*.ts`, but both fail to find some files in `integration_tests` or in `examples`.